README.md

```diff
@@ -251,46 +251,35 @@ Please ensure you have the required [CUDA dependency](#cuda-dependency).
+After extraction of such a pack, you'll find the following contents:
+ - the `*.tsv` files output by CorporaCreator for the downloaded language
+ - the mp3 audio files they reference in a `clips` sub-directory.
 
-We provide an importer (`bin/import_cv2.py`) which automates preparation of the Common Voice (v2.0) corpus as such:
+For bringing this data into a form that DeepSpeech understands, you have to run the CommonVoice v2.0 importer (`bin/import_cv2.py`):
 
 ```bash
-bin/import_cv2.py /path/to/audio/data_dir /path/to/tsv_dir
+bin/import_cv2.py --filter_alphabet path/to/some/alphabet.txt /path/to/extracted/language/archive
 ```
 
-You should have already downloaded the Common Voice v2.0 data from [here](https://voice.mozilla.org/data). The `import_cv2.py` script assumes as input (1) the audio downloaded from Common Voice v2.0 for a certain language, in addition to (2) the `*.tsv` files output by CorporaCreator (included in Common Voice v2.0 download). As output, the script returns the data and transcripts in a state usable by `DeepSpeech.py` (i.e. `*.csv` and `.WAV` data).
+Providing a filter alphabet is optional. It will exclude all samples whose transcripts contain characters not in the specified alphabet. 
+Running the importer with `-h` will show you some additional options.
 
-Please be aware that training with the Common Voice corpus archive requires a lot of free disk space and quite some time to conclude. As this process creates a huge number of small files, using an SSD drive is highly recommended. If the import script gets interrupted, it will try to continue from where it stopped the next time you run it. Unfortunately, there are some cases where it will need to start over. Once the import is done, the directory will contain a bunch of CSV files.
+Once the import is done, the `clips` sub-directory will contain for each required `.mp3` an additional `.wav` file.
+It will also add the following `.csv` files:
 
-The following files are official user-validated sets for training, validating and testing:
+- `clips/train.csv`
+- `clips/dev.csv`
+- `clips/test.csv`
 
-- `cv-valid-train.csv`
-- `cv-valid-dev.csv`
-- `cv-valid-test.csv`
-
-The following files are the non-validated unofficial sets for training, validating and testing:
-
-- `cv-other-train.csv`
-- `cv-other-dev.csv`
-- `cv-other-test.csv`
-
-`cv-invalid.csv` contains all samples that users flagged as invalid.
-
-A sub-directory called `cv_corpus_{version}` contains the mp3 and wav files that were extracted from an archive named `cv_corpus_{version}.tar.gz`.
-All entries in the CSV files refer to their samples by absolute paths. So moving this sub-directory would require another import or tweaking the CSV files accordingly.
+All entries in these CSV files refer to their samples by absolute paths. So moving this sub-directory would require another import or tweaking the CSV files accordingly.
```

Comment:
Why does the importer do this and not use relative paths? As far as I know Deep Speech allows for relative paths.